build.sbt

```diff
@@ -248,19 +188,28 @@ lazy val freestyleLoggingJS  = freestyleLogging.js
 lazy val tests = (project in file("tests")).
   dependsOn(freestyleJVM).
   dependsOn(freestyleMonixJVM).
-  settings(commonSettings: _*).
   settings(noPublishSettings: _*).
   settings(
     libraryDependencies ++= Seq(
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-      "org.scalatest" %%% "scalatest" % "3.0.1" % "test"
-    )
+      "org.scalatest" %%% "scalatest" % "3.0.1" % "test",
+      "org.ensime" %% "pcplod" % "1.2.0" % "test"
+    ),
+    fork in Test := true,
+    javaOptions in Test ++= {
+      val options = (scalacOptions in Test).value.distinct.mkString(",")
+      val cp = (fullClasspath in Test).value.map(_.data).filter(_.exists()).distinct.mkString(",")
+      Seq(
+        s"""-Dpcplod.settings=$options""",
+        s"""-Dpcplod.classpath=$cp"""
+      )
+    },
+    dependencyOverrides += "org.scala-lang" % "scala-compiler" % scalaVersion.value
```

Comment:
yeah, that looks like a dupe.

You need way more `dependencyOverrides` than you'd think, because Ivy is dumb.